kpatch-build/kpatch-build

```diff
@@ -815,6 +818,14 @@ if grep -q "CONFIG_DEBUG_INFO_BTF=y" "$CONFIGFILE" ; then
 	sed -i 's/CONFIG_DEBUG_INFO_BTF/DISABLED_FOR_KPATCH_BUILD/g' "$SRCDIR"/scripts/link-vmlinux.sh || die
 fi
 
+# CONFIG_DEBUG_INFO_BTF_MODULES invokes pahole for kernel modules, which
+# doesn't work with kpatch-cc.  Disable this command by replacing the line
+# with ": \".
+if grep -q "CONFIG_DEBUG_INFO_BTF_MODULES=y" "$CONFIGFILE" ; then
```

Comment:
Let's get rid of this if and move the below code into `CONFIG_DEBUG_INFO_BTF` if. It will work just as well but will make things a bit cleaner.

Otherwise looks good, thank you.